### DISCUSSION TITLE
Payment requests should show the paid addresses at least in print view

### DISCUSSION THREAD

**Discussion by NicolasDorier** [btcpayserver/btcpayserver#2734] [General]
Payment requests doesn't show the address which received the Bitcoins.

This is problematic. If you pay 10 persons in a single transaction and print the payment request page, then the accountant is at loss to reconciliate the outputs of the transaction to the person it actually paid.

**Comment by bolatovumar** [comment]
@NicolasDorier do you "pull payments" instead of "payment requests". I'm not sure how you would pay 10 people with payment requests.

**Reply by NicolasDorier** [reply]
I mean payment requests.
When I pay somebody with a payment request, I can't print a receipt with the address written on it.

**Comment by pavlenex** [comment]
Not sure I understand it either.

@NicolasDorier do you mean we should show an address when the payment request is paid (we show transaciton ID)

![Screenshot 2021-08-02 at 18 19 36](https://user-images.githubusercontent.com/36959754/127892893-207bac61-3b9a-44f5-a999-acd7836659ee.png)

Nevertheless, there's def lots of improvements we can make to the page, right now it provides way too much information for an average user, perhaps we can just show the basics, what's really been paid and show the rest on expansion. IMO `Payment history` should contain payments, not failed payments, especially if we consider this a page that people could print out and hold as an invoice of some sort or a proof for the accountant.

**Reply by dstrukt** [reply]
Second the showing only successful payments on the user facing view.

**Reply by NicolasDorier** [reply]
yes that's what I mean pav

**Reply by dennisreimann** [reply]
Turned this into #2747 and will work on this.
